package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@rsksmart/rif-relay-client",
-  "version": "1.0.0-alpha.1",
+  "version": "1.0.0-alpha.2",
```

If it is a hot fix should it not be 
```suggestion
  "version": "1.0.1-alpha.1",
```
?

---

no "stable" version has been released, so I think we should still move forward the next pre-release number instead of the minor one

---

ok